orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/job/RunJobStage.groovy

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.pipeline.job
+
+import com.netflix.spinnaker.orca.clouddriver.tasks.MonitorKatoTask
+import com.netflix.spinnaker.orca.clouddriver.tasks.job.JobForceCacheRefreshTask
+import com.netflix.spinnaker.orca.clouddriver.tasks.job.RunJobTask
+import com.netflix.spinnaker.orca.pipeline.LinearStage
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import org.springframework.batch.core.Step
+import org.springframework.stereotype.Component
+
+@Component
+class RunJobStage extends LinearStage {
+  public static final String PIPELINE_CONFIG_TYPE = "runJob"
+
+  RunJobStage() {
+    super(PIPELINE_CONFIG_TYPE)
+  }
+
+  @Override
+  List<Step> buildSteps(Stage stage) {
```

Comment:
I'd rather wait to make those changes when they are necessary, since I can't guess now how other cloud providers will implement this stage. 

Having thought about it, I think this stage should block until the Job either succeeds or fails, since a Job is expected to terminate and its result is valuable (i.e. exit code, data on disk, etc...). That's another departure from `CreateServerGroupStage`, and makes combining the two tricky to reason about.